test/test_distributed.py

```diff
@@ -738,8 +737,11 @@ def _spawn_process(self, rank):
         def _run(self, rank):
             self.rank = rank
             try:
-                dist.init_process_group(init_method=INIT_METHOD, backend=BACKEND, world_size=int(WORLD_SIZE))
+                dist.init_process_group(init_method=INIT_METHOD,
+                                        backend=BACKEND,
+                                        world_size=int(WORLD_SIZE))
             except RuntimeError as e:
+                print('Error initializing process group', e)
```

Comment:
Ugh, we shouldn't skip on errors. Can you remove this print and add `raise` after the if below